docs/config.md

```diff
@@ -135,14 +135,14 @@ due to lack of stable semantic convention.
-| `NETRUNTIME`  | [OpenTelemetry.Instrumentation.Runtime](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.Runtime)                                                                                   | *                  | source               | [Experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md) |
-| `PROCESS`     | [OpenTelemetry.Instrumentation.Process](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.Process)                                                                                   | *                  | source               | [Experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md) |
-| `NSERVICEBUS` | [NServiceBus](https://www.nuget.org/packages/NServiceBus)                                                                                                                                       | ≥8.0.0             | source & bytecode    | [Experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md) |
+| ID            | Instrumented library                                                                                                                                                                            | Documentation                                                                                                                                                            | Supported versions | Instrumentation type | Status                                                                                                                            |
+|---------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------|----------------------|-----------------------------------------------------------------------------------------------------------------------------------|
+| `ASPNET`      | ASP.NET Framework \[1\] **Not supported on .NET**                                                                                                                                               | [ASP.NET metrics](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/src/OpenTelemetry.Instrumentation.AspNet/README.md#list-of-metrics-produced)  | *                  | source               | [Experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md) |
```

Comment:
Can we pin the hyperlink to tags? Otherwise it won't be easy to find metrics for previous releases or even the latest release if there would be "in-development" changes in `main`.

The same comment applies to all hyperlinks.

Reply:
Addressed via https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/2359/commits/9e5c3fc813cb4e6fcc252551cac3d187c619085a

The tags do not contain these docs at the moment.

Reply:
Discussed offline. It is not possible right now. Part of the documentation was created yesterday so it is not available on released tags.
releasing.md extended to update links.